app/services/world_location_base_path.rb

```diff
@@ -0,0 +1,20 @@
+class WorldLocationBasePath
+  EXCEPTIONAL_CASES = {
```

Comment:
Can we add a comment indicating the temporary nature of this. Keen for this not to hang around too long.